### PR TITLE
fix: remove empty spaces from ens- or walletAddress input (LF-3268)

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-3268

### CODE REVIEW

File: packages/widget/src/components/SendToWallet/SendToWallet.tsx [hunk 1]
```diff
@@ -31,6 +31,9 @@ export const SendToWallet: React.FC<BoxProps> = forwardRef((props, ref) => {
     rules: {
       required:
         requiredToAddress && (t('error.title.walletAddressRequired') as string),
+      onChange: (e) => {
+        setValue(FormKey.ToAddress, e.target.value.trim());
```
Comment (Abhikumar98) [inline packages/widget/src/components/SendToWallet/SendToWallet.tsx:35]: This looks solid, but I have an very peculiar edge case where the user might paste an ens with a space in middle. To solve for this, we can use a regex to remove all spaces, as `.trim()` also uses a regex afaik internally.

This should work the best, imo
`string.replace(/\s/g, '')`

Comment (chybisov) [reply packages/widget/src/components/SendToWallet/SendToWallet.tsx:35]: Space in the middle? 👀

Comment (Abhikumar98) [reply packages/widget/src/components/SendToWallet/SendToWallet.tsx:35]: haha, yeah, not super peculiar edge case, only possible if the user "*wants-wants*" to do it.

Comment (chybisov) [reply packages/widget/src/components/SendToWallet/SendToWallet.tsx:35]: I mean, are spaces allowed in ENS?

Comment (0xsebn) [reply packages/widget/src/components/SendToWallet/SendToWallet.tsx:35]: > I mean, are spaces allowed in ENS?

Don't believe they are

**Review by chybisov — APPROVED** [review]

**Review by Abhikumar98 — APPROVED** [review]

**Pull request merged**